crates/pbs/src/service.rs

```diff
@@ -27,7 +31,17 @@ impl PbsService {
         let addr = state.config.endpoint;
         info!(version = COMMIT_BOOST_VERSION, commit_hash = COMMIT_BOOST_COMMIT, ?addr, chain =? state.config.chain, "starting PBS service");
 
-        let app = create_app_router::<S, A>(RwLock::new(state).into());
+        // Check if refreshing registry muxes is required
+        let registry_refresh_time = state.config.pbs_config.mux_registry_refresh_interval_seconds;
+        let mut is_refreshing_required = false;
+        if let Some(muxes) = &state.config.registry_muxes {
+            is_refreshing_required = muxes.iter().any(|(loader, _)| {
+                matches!(loader, MuxKeysLoader::Registry { enable_refreshing: true, .. })
+            });
+        }
```

Comment:
a bit more idiomatic
```suggestion
        llet is_refreshing_required = state.config.registry_muxes.as_ref().is_some_and(|muxes| {
            muxes.iter().any(|(loader, _)| {
                matches!(loader, MuxKeysLoader::Registry { enable_refreshing: true, .. })
            })
        });
```